pilot/pkg/kube/inject/monitoring.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inject
+
+import (
+	"istio.io/istio/pilot/pkg/monitoring"
```

Comment:
instead of using `pilot/pkg/monitoring`, can we use `istio/pkg/monitoring`?  that is the replacement. there is an open issue to remove `pilot/pkg/monitoring` from the codebase completely.